alerts/forward.libsonnet

```diff
@@ -51,6 +51,32 @@
               message: 'CoreDNS is returning SERVFAIL for {{ $value | humanizePercentage }} of forward requests to {{ $labels.to }}.',
             },
           },
+          {
+            alert: 'CoreDNSForwardHealthcheckFailureCount',
+            expr: |||
+              rate(coredns_forward_healthcheck_failure_count_total[10s]) > 3
+            ||| % $._config,
+            'for': '30s',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: '{{ $value }} CoreDNS health checks are failing to upstream server {{ $labels.to }} on {{ $labels.instance }}',
+            },
+          },
+          {
+            alert: 'CoreDNSForwardHealthcheckBrokenCount',
+            expr: |||
+              rate(coredns_forward_healthcheck_broken_count_total[5s]) > 3
+            ||| % $._config,
+            'for': '15s',
```

Comment:
Same as above, 10 minutes or 15 minutes for cause is more common.

Reply:
Same as https://github.com/povilasv/coredns-mixin/pull/6#discussion_r470114108

Reply:
Commit `5cbc692` addresses your comment here.